load-test/check-submit-proxy/.vscode/launch.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This vscode dir should probably not be checked in.

Reply:
its optional.  the vscode team do check it in, so they can share debugger config etc.
there is nothing in here that is user specific, it provides support for the azure functions tooling.

i'd be happy with `.idea` and `.vscode` folders, as long as they only include settings that are useful & non-invasive to other team members.

not fussed either way

Reply:
I did caveat it with "probably".  If you intentionally checked it in then by all means leave it.